lib/flipper/model/active_record.rb

```diff
@@ -7,6 +7,25 @@ module ActiveRecord
       def flipper_properties
         {"type" => self.class.name}.merge(attributes)
       end
+
+      # Check if a feature is enabled for this record or any associated actors
+      # returned by `#flipper_actors`
+      def flipper_enabled?(feature_name)
```

Comment:
One interesting thing with this, you could have user and org. In user you override flipper_actors but in org you can't because you don't know the specific user. I think that is part of my concern with adding flipper_actors and this method name (even though it is now scoped). 

I want the convenience but also don't want it to be too *sharp* for users.

Reply:
> One interesting thing with this, you could have user and org. In user you override flipper_actors but in org you can't because you don't know the specific user.

I'm not sure I understand. If you have access to a user, you would check against that instead of the organization.

```ruby
current_user.flipper_enabled?(:thing)
```

If not, then you check against just the organization:

```ruby
current_org.flipper_enabled?(:thing)
```

Either way, enabling the feature for the organization would make both checks pass:

```ruby
Flipper.enable(:thing, current_org)
```

Every feature check would benefit from the user knowing it's related actors.

> I want the convenience but also don't want it to be too _sharp_ for users.

By default the behavior doesn't change. It's only if you explicitly overwrite `#flipper_actors`.

Reply:
If the feature is enable for 1 of the users organizations, then wouldn't they see it enabled on all of their organizations? That doesn't seem correct.